api/aladin/views.py

```diff
@@ -0,0 +1,23 @@
+import django_filters
+from rest_framework import viewsets
+from rest_framework import filters
+from .models import Image
+from .serializers import ImageSerializer
+
+
+class ImageFilter(django_filters.FilterSet):
```

Comment:
Exemplo https://github.com/linea-it/dri/blob/export_targets/api/catalog/views.py#L43
ai não precisa do atributo FilterClass nem Filter Backend pq já está definido um FilterBackend default para todos no defauts.py